app/assets/stylesheets/components/_published-dates.scss

```diff
@@ -1,5 +1,13 @@
 .app-c-published-dates {
   @include core-16;
+
+  [data-controls] {
```

Comment:
is there a reason why you cannot target a class?

Reply:
No, good point. I'll change it now 👍

Reply:
@DilwoarH Done 🙂